src/murmur/Server.cpp

```diff
@@ -465,9 +430,7 @@ void Server::readParams() {
 	bCertRequired      = getConf("certrequired", bCertRequired).toBool();
 	bForceExternalAuth = getConf("forceExternalAuth", bForceExternalAuth).toBool();
 
-	qvSuggestVersion = normalizeSuggestVersion(getConf("suggestversion", qvSuggestVersion));
-	if (qvSuggestVersion.toUInt() == 0)
-		qvSuggestVersion = QVariant();
+	m_suggestVersion = Version::getRaw(getConf("suggestversion", m_suggestVersion != Version::UNKNOWN ? Version::toString(m_suggestVersion) : QString()).toString());
```

Comment:
Maybe for readability's sake it would indeed be better to stick to using a wrapper function for reading this config value. Then one can properly comment on what's going on :eyes:

Reply:
Yes, but you have the magic ability to comment on the exact code snippet I am currently pushing (and working on) ;)
I do not want to compile on my machine directly, so my workflow is: "Commit code" -> "Push" -> "Use my docker container to pull a certain branch and compile". This means that I will sometimes push code which does not compile and I have to first figure out what the problem is. Sorry, if that somehow spams your notifications or something.

Reply:
Haha okay ^^
No worries - My notifications don't blow up because of this :sweat_smile: